Update GraphQL tutorial to use customerCart query (#6585)
magento/devdocs: src/guides/v2.3/graphql/tutorials/checkout/checkout-shopping-cart.md

contributor_link: https://www.atwix.com/
---

> The `customerCart` query returns the active cart for the logged-in customer. If the cart does not exist, the query creates one. The customer’s authorization token must be specified in the headers. ["Get customer authorization token"]({{ page.baseurl }}/graphql/get-customer-authorization-token.html) describes describes these tokens.

Reviewer: ```suggestion
The `customerCart` query returns the active cart for the logged-in customer. If the cart does not exist, the query creates one if you have specified the customer’s authorization token in the headers. ["Get customer authorization token"]({{ page.baseurl }}/graphql/get-customer-authorization-token.html) describes these tokens.
```

Reviewer: what happens if you don't specify the token in the headers? Magento returns error code...?
